db/seeds/support/person_duplicate_seeder.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2020, CVP Schweiz. This file is part of
+#  hitobito and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito.
+
+class PersonDuplicateSeeder
+
+  attr_accessor :encrypted_password
+
+  def initialize
+    @encrypted_password = BCrypt::Password.create("hito42bito", cost: 1)
+  end
+
+  def seed_duplicates
+    candidates.each do |p|
+      group = Group.order('RAND()').first
```

Comment:
Gäbe es eine Möglichkeit das wenigstens deterministisch zufällig zu machen? Also zufällig, aber mit fixem Random-Seed, sodass es bei jedem Seeding reproduzierbar gleich rauskommt?

Reply:
spielt das eine rolle? auf der obersten ebene siehst du ja eh alle ?!

Reply:
Ja schon, aber wenn ich ein Problem finde möchte ich das ja reproduzieren können